util/options.py

```diff
@@ -1,31 +1,123 @@
-import json
 import os
-
+from pydantic import BaseModel, SecretStr, constr
+from pydantic_settings import BaseSettings
 import yaml
+import json
+import subprocess
+from functools import lru_cache
+
+#TODO
+# 1. Add constr to strings to enforce length and format
+# 2. Test Bitwarden overwrites
+
+yaml_settings = dict()
+
+here = os.path.abspath(os.path.dirname(__file__))
+with open(os.path.join(here, "../config/options.yml")) as f:
+    yaml_settings.update(yaml.load(f, Loader=yaml.FullLoader))
+def parse_json_to_dict(json_string):
+    data = json.loads(json_string)
+    return {item['key']: item['value'] for item in data}
+
+try:
+    command = f"bws secret list {yaml_settings['bws']['project_id']}"
```

Comment:
All this stuff including direct process calls to bitwarden should probably be moved into a function for cases where the executor isn't using bitwarden for example for testing the program locally.